qa/tasks/deepsea.py

```diff
@@ -851,36 +856,73 @@ def _run_orch(self, orch_tuple):
                 cmd_str += ' 2>/dev/null'
         if self.dev_env:
             cmd_str = 'DEV_ENV=true ' + cmd_str
-        try:
-            self.master_remote.run(args=[
-                'sudo', 'bash', '-c', cmd_str,
-                ])
-        except CommandFailedError:
-            self.log.error(anchored(
-                "orchestration {} failed. ".format(orch_spec)
-                + "Here comes journalctl!"
-                ))
-            self.master_remote.run(args=[
-                'sudo',
-                'journalctl',
-                '--all',
-                ])
-            raise
-        self.log.info("orchestration {} completed".format(orch_spec))
+        tries = 0
+        if self.survive_reboots:
+            tries = 5
+        self._exec(cmd_str, orch_spec, tries=tries)
+
+    def _exec(self, cmd_str, orch_spec, tries=0):
+        """
+        Execute cmd_str and catch CommandFailedError and
+        ConnectionLostError and retry to run the command
+        until one of the conditons are fulfilled:
+        1) Execution succeeded
+        2) Attempts are exceeded
+        3) CommandFailedError is raised
+        """
+        with safe_while(sleep=60, tries=tries, action="wait for reconnect") as proceed:
```

Comment:
flake8 says this line is too long - please shorten

Reply:
should we have automated flake8 checks in this repo if these rules are enforced?